doc/api/fs.markdown

```diff
@@ -876,6 +877,12 @@ than replacing it may require a `flags` mode of `r+` rather than the
 default mode `w`. The `defaultEncoding` can be `'utf8'`, `'ascii'`, `binary`,
 or `'base64'`.
 
+If `autoClose` is false, then the file descriptor won't be closed, even if
```

Comment:
Would like to see this reworded a bit to make it absolutely obvious that the default behavior is autoClose = true. Perhaps move the third sentence to the front?